src/envoy/mixer/http_control.cc

```diff
@@ -114,6 +114,23 @@ HttpControl::HttpControl(const std::string& mixer_server,
   ::istio::mixer_client::MixerClientOptions options;
   options.mixer_server = mixer_server;
   mixer_client_ = ::istio::mixer_client::CreateMixerClient(options);
+
+  // Extract quota attributes
+  auto it = config_attributes_.find(::istio::mixer_client::kQuotaName);
```

Comment:
Don't like to use namespace at mixer_client level.  We should move these two variables into Attributes class, then use Attributes namespace.   

Will do that later.